### PR TITLE
Add consulting options page

### PR DESCRIPTION
**Pull request by doueal**
https://github.com/SSWConsulting/SSW.Website/issues/150

I have completed the consulting options page which is a combination of the following subtasks.
- [x] Investigate reusable components (ie the booking form)
- [x] Build the page
- [x] #159
- [x] #160

Notes when building this page:
1. I created a tabs react component that can be reused
2. I created the legacy banner that can be reused

image
Figure: Consulting options page with tabs component and legacy banner

### CODE REVIEW

File: src/pages/consulting-options.tsx [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+                <Tabs>
+                    <TabContent header="Book a SSW Expert" icon="one">
+                        <div>
+                            <p>Whether you want a technical expert or a Scrum Master to get the team into a highly performing unit, we have the experts with expertise in a wide range of the Microsoft technology stack. SSW consultants ensure your project is done right and following best practices. From SharePoint or CRM to ASP.NET MVC or Azure our experts get the job done.</p>
+                            <p>For example:</p>
+                            <ul>
+                                <li><a href="WebsiteDevelopment.aspx">Web and MVC Experts</a></li>
+                                <li><a href="/ssw/Consulting/SharePoint.aspx">SharePoint Expert</a></li>
+                                <li><a href="/ssw/Consulting/MicrosoftCRM.aspx">CRM Expert</a></li>
+                                <li><a href="Default.aspx">More</a></li>
```
Comment (lukecookssw) [inline src/pages/consulting-options.tsx]: Change these hrefs to use Gatsby Link components for any/all pages that we've already migrated.
For any links we haven't migrated, use an absolute URL (ssw.com.au/ssw/...) and add a TODO comment to replace them later.

File: src/pages/consulting-options.tsx [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+                                    </div>
+                                </div>
+                            </div>
+                            <hr></hr>
+                            <p><a href="https://www.ssw.com.au/people/">Full List of Experts</a></p>
+                        </div>
+                    </TabContent>
+                    <TabContent header="Book a SSW Team" icon="two">
+                    <div>
+                        <img alt="" src={scrum_bats} className="right clearfix"></img>
```
Comment (lukecookssw) [inline src/pages/consulting-options.tsx]: Add an alt desc

**Review by lukecookssw — CHANGES_REQUESTED** [review]

File: src/pages/consulting-options.tsx (hunk 1 above)
Comment (doueal) [reply src/pages/consulting-options.tsx]: I have changed one anchor link to a gatsby Link:
<li><Link to="/consulting/web-applications/">Web and MVC Experts</Link></li>

and added a TODO for the rest

File: src/pages/consulting-options.tsx (hunk 2 above)
Comment (doueal) [reply src/pages/consulting-options.tsx]: Added an alt description of "Scrum bats image"

**Review by lukecookssw — APPROVED** [review]

**Pull request merged**